browser_use/browser/context.py

```diff
@@ -1085,13 +1085,55 @@ async def _input_text_element_node(self, element_node: DOMElementNode, text: str
 
 			# Get element properties to determine input method
 			is_contenteditable = await element_handle.get_property('isContentEditable')
+			# Check if element is a textarea
+			tag_name = await element_handle.evaluate('el => el.tagName.toLowerCase()')
+			input_type = await element_handle.evaluate('el => el.type.toLowerCase()')
```

Comment:
```suggestion
			tag_name, input_type = await element_handle.evaluate('el => [el.tagName.toLowerCase(), el.type.toLowerCase()]')
```
every CDP round-trip can add 100ms when browser provider is hosted in a cloud environment, lets try to merge these as much as possible